dyc/methods.py

```diff
@@ -72,21 +72,23 @@ def validate(self, result):
             return False
         name = result.name
         if name not in self.config.get(
-            'ignore', []
+            "ignore", []
         ) and not self.is_first_line_documented(result):
             if (
                 self.filename not in self.already_printed_filepaths
             ):  # Print file of method to document
                 click.echo(
-                    "\n\nIn file {} :\n".format(click.style(self.filename, fg='red'))
+                    "\n\nIn file {} :\n".format(
+                        click.style(re.split("/dyc/", self.filename)[1], fg="red")
```

Comment:
My code change is over here. The rest got formatted by running black on this directory.

Reply:
Were you able to test this?

Reply:
Yes. I ran dyc start. I had a result like the following.
![Screenshot from 2019-07-05 00-36-49](https://user-images.githubusercontent.com/27914041/60684772-424f1f80-9ebd-11e9-89fa-8aab00d84cd9.png)

Reply:
Can you explain why you're splitting dyc in the filename? the filename would be the users file on which they are running the script, this means that it would not contain 'dyc' in their file tree

Reply:
Thank you for taking the time to fix this bug btw

Reply:
> Can you explain why you're splitting dyc in the filename? the filename would be the users file on which they are running the script, this means that it would not contain 'dyc' in their file tree

Oh right. I am sorry. I misunderstood eariler.